docs/az-aro-python-development.md

```diff
@@ -104,15 +113,8 @@ for testing. When submitting pull requests, tests that include `azdev style`,
 `azdev linter`, and `azdev test aro` should pass to ensure that changes will not
 impact CI pipelines.
 
-### Reviews
-
-You can use the following command (or similar) to generate a code diff:
-
-```
-diff --color=always -uNr -x '*.pyc' -x vendored_sdks -x tests \
-    ~/src/azure-cli/src/azure-cli/azure/cli/command_modules/aro/ \
-    ~/go/src/github.com/Azure/ARO-RP/python/az/aro/azext_aro/
-```
+Additional instructions for merging changes upstream are contained in internal
```

Comment:
Is there ever a scenario where an external party may need access to these instructions? Asking for the sake of open-ness.

Reply:
I do believe there is no need. Should anyone need to provide a change, they can put a PR here, we review test with our infrastructure and verify it. One it is in our code, it is on us to put it upstream.

Reply:
This is correct. Our internal documentation page is primarily concerned with responsibly utilizing our internal Azure subscription for testing, and what steps are necessary for first-party ARO contributions to our own Azure command module. 

External parties looking to contribute to the Azure CLI, whether to the ARO command module or to others, can follow the upstream Azure CLI's documentation to do so: https://github.com/Azure/azure-cli